cli/determined_cli/tunnel.py

```diff
@@ -108,7 +108,11 @@ def http_connect_tunnel(
     assert parsed_master.hostname is not None, "Failed to parse master address: {}".format(master)
 
     if parsed_master.scheme == "https":
-        context = ssl.create_default_context(cafile=cert_file)
+        if cert_file == "False":
+            # The default constructor does no verification.
+            context = ssl.SSLContext()
```

Comment:
From the [ssl module docs](https://docs.python.org/3/library/ssl.html#ssl.create_default_context):

> The settings are chosen by the ssl module, and usually represent a higher security level than when calling the SSLContext constructor directly

Can we instead use `create_default_context` and manually downgrade `verify_mode` to `CERT_NONE` analgous to [this code example](https://docs.python.org/3/library/ssl.html#ssl.SSLContext.check_hostname)?

Reply:
Okay, done.